sdk/loadtestservice/azure-mgmt-loadtesting/README.md

```diff
@@ -45,6 +45,75 @@ client = LoadTestMgmtClient(credential=DefaultAzureCredential(), subscription_id
 
 ## Examples
 
+### Create a new Azure Load Testing resource
+```python
+from azure.mgmt.loadtesting.models import (
```

Comment:
@venkatr21 README.md of mgmt package is rendered with template automatically so your change will be overrided. You can add the samples in https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/loadtestservice and users can find them in README.md

Reply:
@msyyc Sure, I have raised the samples here: https://github.com/Azure-Samples/azure-samples-python-management/pull/120
Should I undo the readme changes here?

Reply:
@venkatr21 I merged https://github.com/Azure-Samples/azure-samples-python-management/pull/120 and please revert your change about README.md

Reply:
@msyyc I have reverted the readme changes which I have made